src/_tutorials/server/cmdline.md

```diff
@@ -560,36 +500,31 @@ This tutorial described some basic API found in these classes from the dart:io l
-| <a href="{{site.dart_api}}/{{site.data.pkg-vers.SDK.channel}}/dart-io/FileSystemEntity-class.html" target="_blank">FileSystemEntity</a> | Superclass for File and Directory |
-| <a href="{{site.dart_api}}/{{site.data.pkg-vers.SDK.channel}}/dart-io/Platform-class.html" target="_blank">Platform</a> | Provides information about the machine and operating system |
-| <a href="{{site.dart_api}}/{{site.data.pkg-vers.SDK.channel}}/dart-io/stdout.html" target="_blank">stdout</a> | The standard output |
-| <a href="{{site.dart_api}}/{{site.data.pkg-vers.SDK.channel}}/dart-io/stderr.html" target="_blank">stderr</a> | The standard error |
-| <a href="{{site.dart_api}}/{{site.data.pkg-vers.SDK.channel}}/dart-io/stdin.html" target="_blank">stdin</a> | The standard input |
-| <a href="{{site.dart_api}}/{{site.data.pkg-vers.SDK.channel}}/dart-io/exitCode.html" target="_blank">exitCode</a> | Sets the exit code |
-| <a href="{{site.dart_api}}/{{site.data.pkg-vers.SDK.channel}}/dart-io/exit.html" target="_blank">exit()</a> | Sets the exit code and quits |
+| [IOSink]({{ioAPI}}/IOSink-class.html){:target="_blank"} | Helper class for objects that consume data from streams. |
+| [File]({{ioAPI}}/File-class.html){:target="_blank"} | Represents a file on the native file system |
+| [Directory]({{ioAPI}}/Directory-class.html){:target="_blank"} | Represents a directory on the native file system |
+| [FileSystemEntity]({{ioAPI}}/FileSystemEntity-class.html){:target="_blank"} | Superclass for File and Directory |
+| [Platform]({{ioAPI}}/Platform-class.html){:target="_blank"} | Provides information about the machine and operating system |
+| [stdout]({{ioAPI}}/stdout.html){:target="_blank"} | The standard output |
+| [stderr]({{ioAPI}}/stderr.html){:target="_blank"} | The standard error |
+| [stdin]({{ioAPI}}/stdin.html){:target="_blank"} | The standard input |
+| [exitCode]({{ioAPI}}/exitCode.html){:target="_blank"} | Sets the exit code |
+| [exit()]({{ioAPI}}/exit.html){:target="_blank"} | Sets the exit code and quits |
 {: .table }
 
 In addition, this tutorial covers two classes that help with command-line arguments:
-
-| Class | Description |
-|---|---|
-| <a href="{{site.pub-api}}/args/latest/args/ArgParser-class.html" target="_blank">ArgParser</a> | A class that transforms a list of raw arguments and into a set of options, flags, and remaining values. |
-| <a href="{{site.pub-api}}/args/latest/args/ArgResults-class.html" target="_blank">ArgResults</a> | The result of parsing raw command line arguments using ArgParser. |
-{: .table }
+[ArgParser]({{argsAPI}}/ArgParser-class.html){:target="_blank"} and
+[ArgResults]({{argsAPI}}/ArgResults-class.html){:target="_blank"}.
 
 ## Other resources
 
-Check out the [Servers with Dart](https://dart-lang.github.io/server/)
+Check out the [Servers with Dart](https://dart-lang.github.io/server)
 to find more resources related to writing command-line apps.
 
 Refer to the API docs for
```

Comment:
Switch this around ("For more ..., refer to the API reference for...")

(https://developers.google.com/style/clause-order)